ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/HiveRelOptUtil.java

```diff
@@ -77,6 +77,7 @@
 import org.apache.calcite.util.mapping.Mappings;
 import org.apache.hadoop.hive.ql.exec.FunctionRegistry;
 import org.apache.hadoop.hive.ql.optimizer.calcite.reloperators.HiveProject;
+import org.apache.hadoop.hive.ql.optimizer.calcite.rules.HivePointLookupOptimizerRule;
```

Comment:
It's strange to have a utility class depend on a rule. If this method is useful outside the rule then consider moving the dependent parts here or extract them to top-level package private classes.

Reply:
Thanks, I will fix this.